tests/test_tree.py

```diff
@@ -302,4 +302,4 @@ def test_ipl3_config():
     tree = Tree(config='ipl3')
     assert 'MWM_ASTRA' in tree.environ['MWM']
     assert 'ipl-3/spectro/astra' in tree.environ['MWM']['MWM_ASTRA']
-    assert tree.paths['astraAllStarBest'] == '$MWM_ASTRA/{v_astra}/summary/astraAllStarBest-{v_astra}.fits'
\ No newline at end of file
+    assert tree.paths['astraAllStarASCAP'] == '$MWM_ASTRA/{v_astra}/summary/astraAllStarASCAP-{v_astra}.fits'
```

Comment:
This is the line that I think is causing the test to fail.

Reply:
Its just a typo

Reply:
I fixed the typo in the ipl3 test and commited.

Reply:
thanks